.github/workflows/create_js_branch.yaml

```diff
@@ -12,23 +12,35 @@ jobs:
   branch:
     runs-on: ubuntu-latest
     env:
-      BRANCH: releases/${{ github.event.inputs.version}}
+      BRANCH: releases/${{ inputs.version }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
+      - name: setup git config
+        run: |
+          git config user.name "GitHub Action"
+          git config user.email "github-action@users.noreply.github.com"
+
       - name: guarantee branch doesn't exist
         run: |
-          if git ls-remote --heads origin ${BRANCH}) ; then 
+          if git ls-remote --exit-code --heads origin ${BRANCH} ; then 
             exit 1
           else
             exit 0
           fi
+          
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
 
       - name: install dependencies
         run: |
+          rm -rf node_modules/
```

Comment:
do we need to remove the node_modules? it doesn't work without?

Reply:
I added it while trying to make ncc work. I guess we won't need it.

Reply:
removed the script.